git-gui/po/README

```diff
@@ -98,30 +97,26 @@ matching msgid lines.  A few tips:
 
    When you have to change the order of words, you can add "<number>$"
    between '%' and the conversion ('s', 'd', etc.) to say "<number>-th
-   parameter to the format string is used at this point".  For example,
+   a parameter to the format string is used at this point".  For example,
    if the original message is like this:
 
-	"Length is %d, Weight is %d"
+    "Length is %d, Weight is %d"
```

Comment:
This is a whitespace change that might, or might not be desirable. My guess is that it is not wanted.